chess/View/BoardView.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  BoardView.swift
+//  chess
+//
+//  Created by Yeliena Khaletska on 28.06.2024.
+//
+
+import UIKit
+
+class BoardView: UIView {
+
+    private static let boardSideCellLength = 8
+    private static let boardCellCount = BoardView.boardSideCellLength * BoardView.boardSideCellLength
+
+    var gridCalculator = Grid(layout: .dimensions(rowCount: BoardView.boardSideCellLength, columnCount: BoardView.boardSideCellLength))
+
+    let boardModel = BoardModel()
+
+    func generateBoard() {
+        self.gridCalculator.cellCount = BoardView.boardCellCount
+        self.gridCalculator.frame = self.bounds
+        self.boardModel.generateBoard()
```

Comment:
1. having same name for view and model might confuse future readers
2. this violates MVC

<img width="632" alt="image" src="https://github.com/khaletska/chess/assets/13601748/16414710-527b-42a9-bada-cb844c4d1b3f">

Reply:
I don't see yet the whole scheme of the game, was just trying to make it draw something. Moved to VC though